serialization/Deflater.hx

```diff
@@ -128,7 +147,7 @@ class Deflater {
   **/
   public var useEnumIndex(default, null) : Bool;
 
-  var thash : StringMap<Array<DeflatedClass>>;
+  var thash : StringMap<Array<Dynamic>>;
```

Comment:
Can we make this an array of <BaseClass>? Array<Dynamic> goes through reflection for field access on some targets. (Not sure why)